### PR TITLE
Documentation for MTM-42250 Configurable Actility Base URL

### PR DESCRIPTION
**Pull request by ashhSAG93**
Updated connectivity settings page acc to new implementation

### CODE REVIEW

File: content/protocol-integration/lora-actility-bundle/configure-credentials.md [hunk 1]
```diff
@@ -23,15 +24,15 @@ The profile ID, username and password are used to retrieve an access token to se
...
 
 <a name="replace-credentials"></a>
-### Replacing account credentials
+### Updating connectivity settings
 
-In order to replace your credentials, click **Replace credentials**.
+In order to update your connectivity settings, click **Update connectivity settings**.
 
-Enter your profile ID, username, password and application EUI (for details on profile ID and application EUI see [Creating new account credentials](#create-new-credentials).
+Enter your actility ThingPark URL, profile ID, username, password and application EUI (for details on actility thingPark URL, profile ID and application EUI see [Creating new account credentials](#create-new-credentials).)
```
Comment (bbyreddy) [inline content/protocol-integration/lora-actility-bundle/configure-credentials.md:34]: - [ ] Actility needs to start with a capital A

File: content/protocol-integration/lora-actility-bundle/configure-credentials.md [hunk 2]
```diff
@@ -9,10 +9,11 @@ Before using LoRa devices with {{< product-c8y-iot >}}, you need to configure yo
 
 ### <a name="create-new-credentials">Creating new account credentials</a>
 
-If you go to **Connectivity** for the first time, you will be asked to provide credentials and application EUI which is used for LoRa device provisioning.
+If you go to **Connectivity** for the first time, you will be asked to provide credentials, actility ThingPark URL and application EUI which is used for LoRa device provisioning.
```
Comment (bbyreddy) [inline content/protocol-integration/lora-actility-bundle/configure-credentials.md:12]: - [ ] change the order to "..provide Actility ThingPark URL, credentials and application EUI "

File: content/protocol-integration/lora-actility-bundle/configure-credentials.md [hunk 3]
```diff
@@ -9,10 +9,11 @@ Before using LoRa devices with {{< product-c8y-iot >}}, you need to configure yo
...
 
 Enter the following information:
 
+- **actility ThingPark URL**: The Actility API URL may change based on the type of the ThingPark Actility account we use (e.g. Community or Enterprise edition). This is an optional field, and the default value is considered if nothing is entered.
```
Comment (bbyreddy) [inline content/protocol-integration/lora-actility-bundle/configure-credentials.md:16]: - [ ] Capital A for Actility.
- [ ] In the sentence " The Actility API URL may change based on the..." you need to use the name of the field displayed in the UI. I thought UI just says Actility URL, if so update this text accordingly.
- [ ] "the type of the ThingPark Actility account we " - Is it Actility ThingPark or ThingPark Actility. Keep it consistent across the doc.

**Review by bbyreddy — COMMENTED** [review]
I am done with my side of the review.

File: content/protocol-integration/lora-actility-bundle/configure-credentials.md [hunk 4]
```diff
@@ -23,15 +24,15 @@ The profile ID, username and password are used to retrieve an access token to se
 
 ![Setting provider credentials](/images/device-protocols/lora-actility/lora-admin-settings.png)
 
-Click **Save**. If everything is okay, there will be a message "Credentials successfully saved".
+Click **Save connectivity settings**. If everything is okay, there will be a message "Credentials successfully saved".
```
[the remark this answers is not in the document]
Comment (ashhSAG93) [reply content/protocol-integration/lora-actility-bundle/configure-credentials.md]: The message is Connectivity settings saved. Made the change

File: content/protocol-integration/lora-actility-bundle/configure-credentials.md [hunk 5]
```diff
@@ -9,10 +9,11 @@ Before using LoRa devices with {{< product-c8y-iot >}}, you need to configure yo
 
 ### <a name="create-new-credentials">Creating new account credentials</a>
 
-If you go to **Connectivity** for the first time, you will be asked to provide credentials and application EUI which is used for LoRa device provisioning.
+If you go to **Connectivity** for the first time, you will be asked to provide Actility ThingPark URL, credentials and application EUI which is used for LoRa device provisioning.
 
 Enter the following information:
 
+- **Actility ThingPark URL**: The Actility ThingPark URL may change based on the type of the Actility ThingPark account we use (e.g. Community or Enterprise edition). This is an optional field, and the default value is considered if nothing is entered.
```
Comment (bbyreddy) [inline content/protocol-integration/lora-actility-bundle/configure-credentials.md:16]: - [ ] "This is an optional field, and the default value is considered if nothing is entered." change this to suggest that we pick the default value from the properties with which Agent is configured.
Something like "This is an optional field, and defaults to the value configured in the agent properties if nothing is entered."

**Comment by MWindel** [comment]
I am done with my language review so far. If there are more changes done to the document, I'll have another look.

**Comment by MWindel** [comment]
@ashhSAG93 who merges this branch? And in which versions? Thanks for the info!

**Review by MWindel — APPROVED** [review]

**Review by miryalarahul — APPROVED** [review]

**Pull request merged**